lte/gateway/python/magma/tests/pylint_tests.py

```diff
@@ -44,10 +44,13 @@ def test_pylint(self):
         )
         excluded_directories = []
         parent_path = os.path.dirname(os.path.dirname(__file__))
+        print("Starting pylint ...")
```

Comment:
Added some logging for testing - and decided to leave it. If pylint is not installed then there is only a very short skipped message (might be better to make the test fail if pylint is not installed?). This logging makes the distinction a little better visible.